nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/clid/BinariesObject.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.rest.management.clid;
+
+import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
+import static javax.ws.rs.core.Response.Status.CONFLICT;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+import org.nuxeo.ecm.core.blob.DocumentBlobManager;
+import org.nuxeo.ecm.core.blob.binary.BinaryManagerStatus;
+import org.nuxeo.ecm.webengine.model.WebObject;
+import org.nuxeo.ecm.webengine.model.impl.DefaultObject;
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * Endpoint to manage the binaries.
+ *
+ * @since 11.1
+ */
+@WebObject(type = "binaries")
+public class BinariesObject extends DefaultObject {
+
+    /**
+     * Marks and Sweep the orphaned binaries.
+     * 
+     * @return the binary manager status
```

Comment:
I think it's worth documenting, here in the Javadoc or in a global README, what the exact format of the output is going to be: an example JSON would probably be enough. Also HTTP codes in normal and error situations.
This also applies to all other endpoints (also it'd be a separate task).

Reply:
yes i agree, it's what i thought i was just waiting for your point of view.

@troger what you think about the make consistency on the using `@Produces(MediaType.APPLICATION_JSON` on our Rest resources, see [here](https://github.com/nuxeo/nuxeo-management-rest-api/pull/15#discussion_r338429952). I think the `Json` type is the default ones (`rest-management-marshallers-contrib.xml`)